src/ol/style/iconstyle.js

```diff
@@ -576,6 +601,38 @@ ol.style.IconImage_.prototype.load = function() {
 
 
 /**
+ * @private
+ */
+ol.style.IconImage_.prototype.replaceColor_ = function() {
+  if (this.color_ === null) {
```

Comment:
Should also return if `this.tainting_` is `true`.
See https://github.com/openlayers/ol3/pull/4457/files#r45316100